src/Form/DataTransformer/ModelToIdPropertyTransformer.php

```diff
@@ -94,7 +94,11 @@ public function reverseTransform($value)
                 continue;
             }
 
-            $collection[] = $this->modelManager->find($this->className, $id);
+            $object = $this->modelManager->find($this->className, $id);
```

Comment:
We could expose `findMultiple($class, $ids)` (or something similar) in the `ModelManagerInterface` in order to avoid multiple transactions here.

Reply:
Can you create an issue for this, it could make sense, I found some places doing this same thing.

Reply:
#6280.